tff_modular/modules/void/code/sub_modules/mob/void_wound.dm

```diff
@@ -0,0 +1,217 @@
+#define WOUND_VOID_STAGE_FALSE_RECOVERY 0
+#define WOUND_VOID_STAGE_INFECTION 1
+#define WOUND_VOID_STAGE_EXPANSION 2
+#define WOUND_VOID_STAGE_DEEPENING 3
+#define WOUND_VOID_STAGE_PENETRATION 4
+
+#define WOUND_VOID_MESSAGE_COOLDOWN 25 SECONDS
+#define WOUND_VOID_ITEM_DROP_COOLDOWN 12 SECONDS
+#define WOUND_VOID_COUGH_COOLDOWN 2 MINUTES
+
+/**
+ * ВНУТРЕННЯ ПУСТОТА - УЗНАЙ ЧТО ТАКОЕ СТРАДАНИЕ.
+ * Прогресирующий ваунд, в пяти стадиях. Что постепенно уничтожает зараженного.
+ */
+
+/datum/wound/inner_void
+	name = "Void Decay"
+	desc = "Inener void consume you."
+	treat_text = "Threatment impossible. Powerfull painkiller recommendet. \n clinical dead cause body determination."
```

Comment:
Threatment impossible. Using of painkillers highly recommended. Patient death is irreversible.